docs/src/main/sphinx/connector/hive-azure.rst

```diff
@@ -12,30 +12,16 @@ Trino supports both ADLS Gen1 and Gen2. With ADLS Gen2 now generally available,
 we recommend using ADLS Gen2. Learn more from `the official documentation
 <https://docs.microsoft.com/en-us/azure/data-lake-store/data-lake-store-overview>`_.
 
-Hive connector configuration
-----------------------------
-
-All configuration for the Azure storage driver is stored in the Hadoop
-``core-site.xml`` configuration file. The path to the file needs to be
-configured in the catalog properties file:
-
-.. code-block:: text
-
-    hive.config.resources=<path_to_hadoop_core-site.xml>
-
-Configuration for Azure Storage credentials
--------------------------------------------
+Hive connector configuration for Azure Storage credentials
+----------------------------------------------------------
 
-If you do not want to rely on Hadoop's ``core-site.xml`` and want to have Trino
-configured independently with the storage credentials, you can use the following
-properties in the catalog configuration.
-
-We suggest to use this kind of configuration when you only have the Primary
-Storage account linked to the cluster. When there are secondary storage
-accounts involved, we recommend configuring Trino using a ``core-site.xml``
-containing the appropriate credentials for each account, as described in the
-preceding section.
+To configure Trino to use the Azure Storage credentials, set the following
+configuration properties in the catalog properties file. It is best to use this
+type of configuration if the primary storage account is linked to the cluster.
 
+The specific configuration depends on the type of storage and uses the
+properties from the following sections in the catalog properties file.
+   
```

Comment:
just add a sentence here that mentions that in more complex cases you can use the core-site xml (and link to that section)..